sdk/go/common/apitype/events.go

```diff
@@ -139,7 +139,7 @@ type StepEventMetadata struct {
 	// Keys that changed with this step.
 	Diffs []string `json:"diffs,omitempty"`
 	// The diff for this step as a list of property paths and difference types.
-	DetailedDiff map[string]PropertyDiff `json:"detailedDiff,omitempty"`
+	DetailedDiff map[string]PropertyDiff `json:"detailedDiff"`
```

Comment:
Should we have a comment here that we explicitly do not want `omitempty`? I'm worried someone might add it back mistakenly.

Reply:
+1 to this

Reply:
added a comment!